packages/suite/src/utils/wallet/cardanoUtils.ts

```diff
@@ -36,8 +36,20 @@ export const getProtocolMagic = (accountSymbol: Account['symbol']) =>
 export const getNetworkId = (accountSymbol: Account['symbol']) =>
     accountSymbol === 'ada' ? CARDANO.NETWORK_IDS.mainnet : CARDANO.NETWORK_IDS.testnet;
 
-export const getAddressType = (accountType: Account['accountType']) =>
-    accountType === 'normal' ? CardanoAddressType.BASE : CardanoAddressType.BYRON;
+export const getDerivationType = (accountType: Network['accountType']) => {
```

Comment:
maybe `cardanoGetDerivationTypeFromAccountType`? I don't know if better. Just at the frist glance `getDerivationType` felt a little bit too general. Feel free to dismiss this one.

Reply:
I agree that we could be more clear it is cardano specific function. hmm maybe then it would be good thing to rename all functions within cardanoUtils to keep it consistent across file? I But that's probably task for another PR.